inst/pages/21_microbiome_community.qmd

```diff
@@ -94,7 +100,8 @@ ggplot(df, aes(x = SampleID, y = FeatureID, fill = clr_z)) +
   labs(x = "Samples", y = "Taxa")
 ```
 
-_pheatmap_ is a package that provides methods to plot clustered heatmaps. 
+*pheatmap* is a package that provides methods to plot clustered
+heatmaps.
```

Comment:
Can we switch to ComplexHeatmap/sechm?

Reply:
This part wasn't done by me - do you still want me to make that change?

Reply:
Yep, you could just  ComplexHeatmap::pheatmap(). That is simple fix

We should limit packages in OMA. ComlexHeatmap does the same than pheatmap and more

Reply:
perfect thanks